haiku-apps/bshisen/bshisen-1.2.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="BeOS tile matching game"
+DESCRIPTION="BShisen is a Majong style game made for Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/BShisen"
+COPYRIGHT="1998-1999 Kelvin W Sherlock"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="1e86e4daec9f48d9aa9642227f962159cfe3a733"
+SOURCE_URI="https://github.com/HaikuArchives/BShisen/archive/$srcGitRev.tar.gz"
+#CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
+SOURCE_FILENAME="BShisen-$srcGitRev.tar.gz"
+SOURCE_DIR="BShisen-$srcGitRev/source"
+
+ARCHITECTURES="?x86_gcc2 ?x86_64"
```

Comment:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86_64"
+ARCHITECTURES="x86_gcc2 !x86_64"
```
On x86_gcc2 the compilation succeeds for BitmapControl.cpp, Prefs.cpp, Tiles.cpp and ID_Win.cpp but fails on main.cpp:
```
g++ -c main.cpp -iquote./  -iquote./   -O1   -Woverloaded-virtual -funsigned-bitfields -Wwrite-strings  -o "objects/main.o"
In file included from Shisen.h:9:0,
                 from main.cpp:14:
YLanguageClass.h:79:7: error: 'map' does not name a type
       map <const char*,const char*,ltstr>LangList;
```
So you can probably just mark x86_64 as broken.